README.md

```diff
@@ -1,6 +1,6 @@
 # Berlin Hack and Tell
 
-[Meetup](https://www.meetup.com/de-DE/Berlin-Hack-and-Tell)
+Last Tuesday on Jan-Nov | [Meetup Page](https://www.meetup.com/de-DE/Berlin-Hack-and-Tell) | [@c-base](https://www.c-base.org)
```

Comment:
I would like Jan..Nov more :-) But then we also really have to make sure there is no more breaks in between - there have been times without BHNT but it was Jan..Nov - cannot compute